### PR TITLE
MOSIP-21020

### PR DESCRIPTION
**Pull request by dhanendra06**

### CODE REVIEW

File: admin/admin-service/src/main/java/io/mosip/admin/controller/ApplicantDetailsController.java [hunk 1]
```diff
@@ -31,11 +34,21 @@ private ResponseWrapper<ApplicantDetailsDto> getApplicantDetails(@PathVariable("
...
...bHjn-N2iw0iLFoGv1LJlv8h8NLk87iwVOYRcbtEa98qCLA6aoIDI6zDhQy56KS9VICgOz7z9IvdynHnJviYxVvPvpmeQLztphXt6nVx0T3dH2kOXAjqIjFksIOFkPIWkxmr2KS9QIC0xODUyL1IgNS9PRSAoZLVOMwss9kuFqysRjMirtejt-Dv5RGyjbt_cWhW9aT4pL1VFICikf0SwGFMmsCKp5C3ibL3EIipcXPexOVljQD46V3Bccj-RKS9QZXJtcyAoI6iW2FphueWHXFzykhKxieUpL1YgNS9MZW5ndGggMjU2L1N0ckYvU3RkQ0YvU3RtRi9TdGRDRi9DRjw8L1N0ZENGPDwvTGVuZ3RoIDMyL0F1dGhFdmVudC9Eb2NPcGVuL0NGTS9BRVNWMz4-Pj4-PgplbmRvYmoKeHJlZgowIDIwCjAwMDAwMDAwMDAgNjU1MzUgZiAKMDAwMDAwMDE5MiAwMDAwMCBuIAowMDAwMDI1NzAxIDAwMDAwIG4gCjAwMDAwMjYzODEgMDAwMDAgbiAKMDAwMDAyNTYxMyAwMDAwMCBuIAowMDAwMDI1OTA1IDAwMDAwIG4gCjAwMDAwMjYxNTUgMDAwMDAgbiAKMDAwMDAwMDAxNSAwMDAwMCBuIAowMDAwMDI3MzY3IDAwMDAwIG4gCjAwMDAwMjU0MzggMDAwMDAgbiAKMDAwMDAyNTUzNiAwMDAwMCBuIAowMDAwMDI2NzE2IDAwMDAwIG4gCjAwMDAwMjcxMzIgMDAwMDAgbiAKMDAwMDAyNzMxNSAwMDAwMCBuIAowMDAwMDI3NTYxIDAwMDAwIG4gCjAwMDAwMjg0NzcgMDAwMDAgbiAKMDAwMDAyODU2OCAwMDAwMCBuIAowMDAwMDI4NjU4IDAwMDAwIG4gCjAwMDAwMzQ2NjkgMDAwMDAgbiAKMDAwMDAzODA4MCAwMDAwMCBuIAp0cmFpbGVyCjw8L1NpemUgMjAvUm9vdCAxMiAwIFIvSW5mbyAxMSAwIFIvRW5jcnlwdCAxOSAwIFIvSUQgWzxjNDQ0OTlmM2QwNWQyMDI4ZDRjODcwN2VmZDUyZWQ1YT48MjlkOWI3MGNkZDU1MGIzYjhkZDU3Nzk1ZjE2ZjQ3Yzc-XT4-CiVpVGV4dC01LjUuMTMKc3RhcnR4cmVmCjM4NDMzCiUlRU9GCg";
```
Comment (ase-101) [inline admin/admin-service/src/main/java/io/mosip/admin/controller/ApplicantDetailsController.java:50]: Why is the data hardcoded here ?

File: admin/admin-service/src/main/java/io/mosip/admin/service/impl/ApplicantDetailServiceImpl.java [hunk 2]
```diff
@@ -129,19 +150,32 @@ public byte[] getRIDDigitalCard(String rid, boolean isAcknowledged) throws Excep
...
         }
-        data = restClient.getForObject(digitalCardStatusResponseDto.getUrl(), String.class);
+      //  data = restClient.getForObject(digitalCardStatusResponseDto.getUrl(), String.class);
         return data.getBytes();
     }
 
+    @Override
+    public ApplicantUserDetailsDto getApplicantUserDetails() {
+        String userId = ((UserDetails) SecurityContextHolder.getContext().getAuthentication().getPrincipal()).getUsername();
+        long count=5;
```
Comment (ase-101) [inline admin/admin-service/src/main/java/io/mosip/admin/service/impl/ApplicantDetailServiceImpl.java:168]: why is this value hardcoded here?

File: admin/admin-service/src/main/java/io/mosip/admin/controller/ApplicantDetailsController.java (hunk 1 above)
Comment (dhanendra06) [reply admin/admin-service/src/main/java/io/mosip/admin/controller/ApplicantDetailsController.java:50]: we have mocked for integration testing

File: admin/admin-service/src/main/java/io/mosip/admin/service/impl/ApplicantDetailServiceImpl.java (hunk 2 above)
Comment (dhanendra06) [reply admin/admin-service/src/main/java/io/mosip/admin/service/impl/ApplicantDetailServiceImpl.java:168]: we have mocked for integration testing

**Pull request merged**